src/backend/access/transam/xlog.c

```diff
@@ -5067,6 +5067,8 @@ CheckRequiredParameterValues(void)
 	}
 }
 
+#define NEON_SIGNAL_FILE		"neon.signal"
+
```

Comment:
Could you rebase this on top of the latest REL_16_STABLE_neon?

I suspect that we already have NEON_SIGNAL_FILE defined in the xlog.h over there.

Reply:
Done, was a re-define but the compiler failed to catch it. Thanks!